src/utils/highlights.ts

```diff
@@ -2,7 +2,7 @@ import { Highlight } from '../types/video-state';
 
 import { PROGRESS_BAR_DIVIDER } from './constants';
 
-interface Segment {
+export interface Segment {
```

Comment:
Pls add a comment what a segment is (esp the difference segment vs highlight)

Reply:
`Highlight` should be an interface extended from `Segment`.
```ts
interface Highlight extends Segment {
  id: string;
  color: string;
}
```
Correcting that +